examples/models/maps_cities.py

```diff
@@ -1,3 +1,12 @@
+''' A plot showing a map of the world, highlighting cities where the population is over 5,000 people, made using the 'GMapPlot' class.
+
+.. bokeh-example-metadata::
+    :sampledata: world_cities
+    :apis: bokeh.models.glyphs.Circle, bokeh.models.map_plots.GMapPlot, bokeh.models.tools.PanTool, bokeh.models.tools.WheelZoomTool
+    :refs: :ref:`_ug_styling_plots_axes_labels`, :ref:`_ug_topics_geo`, :ref:`_ug_basic_data`
```

Comment:
geo ref could be more specific
```suggestion
    :refs: :ref:`_ug_styling_plots_axes_labels`, :ref:`_ug_topics_geo_google_maps`, :ref:`_ug_basic_data`
```

Reply:
Done